### PR TITLE
Aquamarine stuff

### PR DESCRIPTION
**Pull request by vaxerski**
ref https://github.com/hyprwm/Hyprland/pull/6608

### CODE REVIEW

**Comment by ikalco** [comment]
looking at all remaining mentions of _**wlr**_:

pages/Plugins/Development/Advanced.md

>For members, e.g. CCompositor::focusWindow(CWindow\*_**, wlr_surface\***_) you will also need to add the thisptr argument to your hook:

pages/Getting Started/Installation.md
>Lastly, copy hyprctl, hyprpm, and _**wlroots**_ as mentioned
[here](#manual-releases-linux-only)

pages/FAQ/_index.md, (just use `hyprctl monitors`)
>### My monitor no worky
>
>Try changing the mode in your config. If your preferred one doesn’t work, try a lower one. A good way to list all modes is to get _**wlr-randr and do a wlr-randr --dryrun**_

File: pages/Getting Started/Installation.md [hunk 1]
```diff
@@ -211,15 +211,12 @@ copy hyprctl to `/usr/bin/`.
...
-copy the wlroots .so (`libwlroots.so.XX032`) to `/usr/lib/`.
-
 copy the desktop entry (`example/hyprland.desktop`) to
 `/usr/share/wayland-sessions/`
 
 the example config is in `example/hyprland.conf`.
 
-For updating later on, you can overwrite the binaries (hyprctl, hyprland and
-libwlroots), you don't need to update anything else.
+For updating later on, you can overwrite the binaries (hyprctl, hyprland, hyprpm ...), you don't need to update anything else.
```
Comment (fufexan) [inline pages/Getting Started/Installation.md:219]:
```suggestion
For updating later on, you can overwrite the binaries (hyprctl, hyprland, hyprpm ...). You don't need to update anything else.
```

File: pages/Getting Started/Installation.md [hunk 2]
```diff
@@ -251,7 +248,7 @@ zypper in gcc-c++ git meson cmake "pkgconfig(cairo)" "pkgconfig(egl)" "pkgconfig
 
 ```plain
 pkg install git pkgconf gmake gcc evdev-proto cmake wayland-protocols wayland libglvnd libxkbcommon libinput cairo pango pixman libxcb
-pkg install meson jq `pkg rquery %dn wlroots` hwdata libdisplay-info libliftoff
+pkg install meson jq hwdata libdisplay-info libliftoff
```
Comment (fufexan) [inline pages/Getting Started/Installation.md:251]: Isn't aquamarine also needed? Or is it pulled in by the `hyprland` package/port.

**Comment by fufexan** [comment]
> pages/FAQ/_index.md, (just use `hyprctl monitors`)
> 
> > ### My monitor no worky
> > Try changing the mode in your config. If your preferred one doesn’t work, try a lower one. A good way to list all modes is to get _**wlr-randr and do a wlr-randr --dryrun**_

`wlr-randr` can be left there, as it's still supported through the output management protocol.

File: pages/Getting Started/Installation.md (hunk 2 above)
Comment (vaxerski) [reply pages/Getting Started/Installation.md:251]: dossnt have a pkg yet

**Comment by vaxerski** [comment]
rest is ok

**Comment by vaxerski** [comment]
code example doesnt matter that much, its just an example

File: pages/Getting Started/Installation.md [hunk 3]
```diff
@@ -367,7 +361,7 @@ Now, of course, install manually.
 sudo cp ./build/Hyprland /usr/bin && sudo cp ./example/hyprland.desktop /usr/share/wayland-sessions
 ```
 
-Lastly, copy hyprctl, hyprpm, and wlroots as mentioned
+Lastly, copy hyprctl, hyprpm, hyprctl, etc as mentioned
```
Comment (ErynGalen) [inline pages/Getting Started/Installation.md:364]: `hyprctl` is mentionned twice

Comment (vaxerski) [reply pages/Getting Started/Installation.md:364]: thanks

Comment (vaxerski) [reply pages/Getting Started/Installation.md:364]: fixed

**Pull request merged**